src/operators.jl

```diff
@@ -24,9 +24,10 @@ export map,
 Transform signal `s` by applying `f` to each element. For multiple signal arguments, apply `f` elementwise.
 """
 function map(f, input::Signal, inputsrest::Signal...;
```

Comment:
I think because it was causing ambiguity for the `map(f)` method sinc Base defines `map(f, xs...)` too.